packages/solid-query/package.json

```diff
@@ -16,7 +16,7 @@
   "exports": {
     ".": {
       "types": "./build/lib/index.d.ts",
-      "solid": "./build/solid/index.jsx",
+      "solid": "./build/solid/index.js",
```

Comment:
I think we'd now need to change `"sideEffects": false,` here in `package.json` to point to:

```
  "sideEffects": [
    "./src/setBatchUpdatesFn.ts"
  ],
```

right?

Reply:
Yeah I missed this. I'll fix it in a new commit

Reply:
Done!